src/reducers/__tests__/academy.ts

```diff
@@ -0,0 +1,26 @@
+import { IAction, LOG_OUT, SAVE_CANVAS } from '../../actions/actionTypes';
+import { reducer } from '../academy';
+import { defaultAcademy, IAcademyState } from '../states';
+
+test('LOG_OUT works correctly on default academy', () => {
+  const action: IAction = {
+    type: LOG_OUT,
+    payload: {}
+  };
+  const result: IAcademyState = reducer(defaultAcademy, action);
+
+  expect(result).toEqual(defaultAcademy);
+});
+
+test('SAVE_CANVAS works correctly on default academy', () => {
+  const action: IAction = {
+    type: SAVE_CANVAS,
+    payload: null
```

Comment:
We ideally dont want to be using `null` as the payload. The payload for this is of type `HTMLCanvasElement`. 

You can create one using this function below:
```
function createContext(width : number, height : number) : HTMLCanvasElement {
    let canvas = document.createElement('canvas');
    canvas.width = width;
    canvas.height = height;
    return canvas
}
```

Reply:
Changes have been made, thanks Rahul!